peps/pep-0718.rst

```diff
@@ -158,7 +197,14 @@ The following code snippet would fail at runtime without this change as
    def bar[U]():
        return Foo[int]()
 
-   assert bar[str]().__orig_class__  is Foo[int]
+   assert bar[str]().__orig_class__  == Foo[int]
+
+Interactions with `@typing.overload`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Lint fix, two backticks for code formatting in reST (and a newline after the header line):

```suggestion
Interactions with ``@typing.overload``
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

```

Reply:
Ah yes thanks, was wondering why it wasn't linting, have been writing too much markdown recently